test/07_levels.jl

```diff
@@ -112,7 +112,7 @@ module TestLevels
         @test pool.valindex == [CategoricalValue(i, pool) for i in 1:5]
     end
 
-    @test levels!(pool, [1, 2, 3]) == [1, 2, 3]
+    @test levels(levels!(pool, [1, 2, 3])) == [1, 2, 3]
```

Comment:
Should rather test that `levels!` returns `pool` using `===`, just in case. Then on a second line, check that `levels` returns `[1, 2, 3]`.